gap/matrix/slconstr.gi

```diff
@@ -2466,56 +2466,52 @@ end;
 #! @BeginChunk NaturalSL
 #! TODO
 #! @EndChunk
-FindHomMethodsMatrix.NaturalSL := function(ri,G)
-  local data,e,genlist,grpmem,i,j,n,p,q,r,il;
-
+FindHomMethodsMatrix.NaturalSL := function(ri, G)
+    local data, e, genlist, grpmem, i, j, n, p, q, r, il;
     if not IsMatrixGroup(G) then
         return NeverApplicable;
     fi;
-
-    il := InfoLevel(InfoMethSel);
-    SetInfoLevel(InfoMethSel,0);
-    r:=RecogniseClassical(G,"linear");
-    SetInfoLevel(InfoMethSel,il);
+    r := RecogniseClassical(G, "linear");
     if not( (IsBound(r.isSLContained) and r.isSLContained = true) ) then
         return TemporaryFailure;
     fi;
-
-
-  n:=ri!.dimension;
-  q:=Size(ri!.field);
-  p:=Factors(q)[1];
```

Comment:
Are large scaler whitespace and indentation changes really going to help us? They do wreck havoc on `git blame`, though...

Reply:
I was looking through the function and kinda got carried away. But yeah, I should try more not to spend time on changing whitespace only. In my defense, the indentation looked reaally wonky. ^^